_posts/2017-03-18-hypatia-meets-womens-energy-club_en.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Hypatia e.V. meets Women's Energy Club Kyiv
+description: Meeting on 15th / 16th of March
+header: Hypatia e.V. meets Women's Energy Club Kyiv
+---
+
+![PotatoeChild](/img/potatoechild.jpeg "PotatoeChild")
+**Women's Energy Club Ukraine** is a network of women working in energy related sectors. It aims to empower women and provide professional support. The club unites members of the Parliament of Ukraine (Energy Committee), experts, international organizations representatives as well as women from energy production sector. It is the first network for women in Ukraine. Currently the Club has about 30 members. The Club is supported by the European-Ukrainian Energy Agency & Heinrich-Böll-Foundation in Kiev.
+
+[**Hypatia Frauennetzwerk Erneuerbare Energien und Cleantech e.V.**](www.hypatia-network.de) is a non-profit organization for women working in the renewable energy sector. Similar to the Women's Energy Club it aims to empower women and provide professional support. Increasing the visibility of women in the sector is one of the major goals to reach for professional equity for women and men. Hypatia has about 30 members. The organization team works on voluntary basis.
+
+The same objectives led the Women's Energy Club to invite Hypatia in order to present our network & benefit from each other knowledge.  
+As a representative of Hypatia I traveled to the latest meeting of the Club on 15th of March in Kiev. We established that the general problems women face in professional life are the same in Ukraine and Germany.  
+A lot of women are not aware of being discriminated in the professional context. Our networks want to increase awareness and change the established paradigms. This led to the conclusion to **give women the opportunity of choice without any disadvantages in the professional context**.  
+The main conclusions of our discussion are listed below:
+
+*General Problems*
+- Visibility of women in profession
+- Lack of female role-models
+- Inefficient knowledge exchange between women concerning career issues (salary, work conditions...)
+- Equity of men and women in profession (acceptance, opportunity, salary...) 
+
+*Importance of Networking*  
+- Knowledge exchange concerning career & work-life-balance
+- Support for women in male-dominated sectors
+- Open table discussions concerning career and/or related technology topics
+- Conduct workshops and events to support women's career with skills and strategies
```

Comment:
Is this about the importance of networking or are these methods to generate networking-friendly occassions?